blazor/dialog/predefined-dialogs.md

```diff
@@ -0,0 +1,338 @@
+---
+layout: post
+title: Predefined dialog in Blazor Dialog Component | Syncfusion
+description: Checkout and learn here all about Predefined dialog in Syncfusion Blazor Dialog component and much more.
+platform: Blazor
+control: Dialog
+documentation: ug
+---
+
+# Predefined dialogs in Blazor Dialog componen
+
+The dialog component is used to render the `Alert`, `Confirm` and `Prompt` dialogs with the minimal code. The alert, confirm and prompt dialogs are shown using [DialogServices](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Popups.SfDialogService.html).
```

Comment:
The dialog component is used to render the `Alert`, `Confirm`, and `Prompt` dialogs with minimal code. The alert, confirm, and prompt dialogs are shown using [DialogServices](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Popups.SfDialogService.html).